.gitmodules

```diff
@@ -3,4 +3,5 @@
 	url = https://github.com/open-source-parsers/jsoncpp.git
 [submodule "tools/rpc_spec"]
 	path = tools/rpc_spec
-	url = https://github.com/smartdevicelink/rpc_spec.git
+	url = https://github.com/SNiukalov/rpc_spec.git
```

Comment:
@SNiukalov Why do we have your personal repo here?

Reply:
@AByzhynar 
Due to the [latest changes committed by the PM](https://github.com/smartdevicelink/sdl_core/pull/3270/files),
currently to build SDL it is necessary to use MOBILE_API from the https://github.com/smartdevicelink/rpc_spec.

Luxoft team does not have access to create branches in https://github.com/smartdevicelink/rpc_spec repo, but MOBILE_API is used by unit tests.

Therefore, changes to https://github.com/smartdevicelink/rpc_spec repo were created in SNiukalov:feature/removing_url_parameter_max_length branch

Reply:
@SNiukalov Clear.Thank you.